sos/report/plugins/openshift_ovn.py

```diff
@@ -32,6 +32,10 @@ def setup(self):
             "/var/lib/ovn-ic/etc/ovnnb_db.db",
             "/var/lib/ovn-ic/etc/ovnsb_db.db"
         ])
+        # Collect additional ovnkube logs for interconnect setup.
+        self.add_copy_spec([
+            "/var/lib/ovn-ic/etc/*log*"
+        ])
```

Comment:
This can be a one-line addition to the `add_copy_spec()` call above, since we're working in the same directory. Also, a_c_s accepts single strings as well as lists if this were to be kept separate.

Reply:
OK @TurboTurtle , moved this log path into existing list.